spark-indexer/src/main/java/org/uniprot/store/spark/indexer/taxonomy/mapper/TaxonomyStatisticsWrapperJoinMapper.java

```diff
@@ -0,0 +1,45 @@
+package org.uniprot.store.spark.indexer.taxonomy.mapper;
+
+import org.apache.spark.api.java.Optional;
+import org.apache.spark.api.java.function.Function;
+import org.uniprot.core.taxonomy.TaxonomyStatistics;
+import org.uniprot.core.taxonomy.impl.TaxonomyStatisticsBuilder;
+import org.uniprot.store.spark.indexer.taxonomy.mapper.model.TaxonomyStatisticsWrapper;
+
+import scala.Tuple2;
+
+public class TaxonomyStatisticsWrapperJoinMapper
+        implements Function<
+                Tuple2<Optional<TaxonomyStatisticsWrapper>, Optional<TaxonomyStatisticsWrapper>>,
+                TaxonomyStatisticsWrapper> {
+
+    private static final long serialVersionUID = 7240080849019578656L;
+
+    @Override
+    public TaxonomyStatisticsWrapper call(
```

Comment:
after thinking about the class name, I think *Wrapper is fine. It's a wrapper, after all. We could call it something like *Info, but arguably, it's not clear that it wraps a TaxStats object... therefore, I think TaxonomyStatisticsWrapper is good.